lib/fluent/event.rb

```diff
@@ -162,5 +182,21 @@ def to_msgpack_stream
       @data
     end
   end
-end
 
+  module ChunkMessagePackEventStreamer
+    include MessagePackFactory::Mixin
+    # chunk.extend(ChunkEventStreamer)
+    #  => chunk.each{|time, record| ... }
+    def each(&block)
```

Comment:
MessagePack::Unpacker raises EOFError if data underflow occurs when unpacking data. It's caused by broken/unflushed data on disk.
For broken data, this code (without rescue) might raise failure (and it will not be solved forever), but it should be because it's content data of users.
For unflushed data, this code will raise error at once, but retry will succeed if it's after successful flush.

So, this code shouldn't do `rescue` and ignore.